src/target/riscv/riscv-013.c

```diff
@@ -1038,9 +1054,10 @@ static int register_write_abstract(struct target *target, enum gdb_regno number,
 	if (write_abstract_arg(target, 0, value, size) != ERROR_OK)
 		return ERROR_FAIL;
 
-	int result = execute_abstract_command(target, command);
+	unsigned int abstractcs_err;
+	int result = execute_abstract_command(target, command, &abstractcs_err);
```

Comment:
Same as above.

There are more occurrences throughout this file. If you agree with the suggestion, please take a look at all of them. I have not commented on the other cases to keep the number of comments low.

```suggestion
	unsigned int cmderr;
	int result = execute_abstract_command(target, command, &cmderr);
```

Reply:
addressed